packages/paste-website/src/components/site-wrapper/sidebar/SidebarDesktop.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Sidebar, SidebarHeader, SidebarBody, SidebarHeaderIconButton, SidebarHeaderLabel} from '@twilio-paste/sidebar';
+import {LogoTwilioIcon} from '@twilio-paste/icons/esm/LogoTwilioIcon';
+
+import {SidebarNavigation} from './SidebarNavigation';
+
+export interface SidebarDesktopProps {
+  mainContentSkipLinkID: string;
+  sidebarNavigationSkipLinkID: string;
+  topbarSkipLinkID: string;
+}
+
+const SidebarDesktop: React.FC<React.PropsWithChildren<SidebarDesktopProps>> = ({
+  mainContentSkipLinkID,
+  sidebarNavigationSkipLinkID,
+  topbarSkipLinkID,
+}) => {
+  return (
+    <Box display={['none', 'block']}>
```

Comment:
I think we can remove this Box if we remove SidebarMobile and just have the one Sidebar